### ISSUE TITLE
Defining one component started last and stopped first

### ISSUE THREAD

**Issue by dreyling** [eudaq/eudaq#422]
... or order of starting/stopping components.

With that mechanism, the central hardware device (usually the TLU) could be assigned ensuring for a synchronized start and stop process.

Possible implementation
- setting flag in ini or conf file, e.g. for [RunControl] setting the name of the component
- check if all other components are Started
- then start the the component
- correspondingly for stopping process

@DavidCussans @PaoloGB @MengqingWu

**Comment by MengqingWu** [comment]
@dreyling I just found it from the web notification.. It is a good idea to add this, either a flag or so to implement this, may I ask if any recent development done for it? If not, maybe we should talk about this?

**Comment by arummler** [comment]
There was something submitted by Yi to make sure that the DataCollector is started. I mentioned in one of our meetings or in some issue that I cannot find in the moment that a full fledged start order might be interesting. Either as dreyling wrote with dependencies and/or like WinCC it does with ranks (assignment of number from 0 to 99) where first all processes of one rank need to be finished before the others are started.

**Comment by dreyling** [comment]
Yes, this is implemented already using this parameter of the RunControl and set the tag-name (option - t tag-name when starting the producer):https://github.com/eudaq/eudaq/blob/master/user/eudet/misc/telescope_autotrigger_eudet.conf

**Comment by arummler** [comment]
@dreyling In the moment the scope is quite limited. It would be useful to make sure that e.g. multiple montors are only activated after their corresponding producers are already up and running.

**Comment by dreyling** [comment]
@arummler the origin for doing this was that the TLU is the last instance. This is done. 
For an extension, I suggest to open a new issue describing the use case, which is not handled correctly.

**Comment by MengqingWu** [comment]
Great to know that this has been done already! I noted down to check and see if any coherent change I need for our modules.
